# HG changeset patch
# User jwe
# Date 749793757 0
#      Tue Oct 05 04:02:37 1993 +0000
# Node ID b891106886252f6c2963282bb77ca56f88c1cd88
# Parent  edfa5a96c5f159cc6992487cd9ab16d9f9e1c4b5
[project @ 1993-10-05 04:02:37 by jwe]
(DO_COMMA_INSERT_CHECK): Delete unused macro.

diff --git a/src/lex.h b/src/lex.h
--- a/src/lex.h
+++ b/src/lex.h
@@ -41,18 +41,16 @@ Software Foundation, 675 Mass Ave, Cambr
   do \
     { \
       error (msg); \
       jump_to_top_level (); \
     } \
   while (0)
 #endif
 
-#define DO_COMMA_INSERT_CHECK yyless (do_comma_insert_check ())
-
 #define TOK_RETURN(tok) \
   do \
     { \
       current_input_column += yyleng; \
       quote_is_transpose = 0; \
       cant_be_identifier = 0; \
       convert_spaces_to_comma = 1; \
       return (tok); \
